Library/Formula/sourcekitten.rb

```diff
@@ -0,0 +1,16 @@
+class SourceKitten < Formula
+  homepage "https://github.com/jpsim/SourceKitten"
+  url "https://github.com/jpsim/SourceKitten.git", :tag => "0.3.0"
+  head "https://github.com/jpsim/SourceKitten.git"
+
+  depends_on :xcode => ["6.1.1", :build]
+
+  def install
+    system "make", "prefix_install", "PREFIX=#{prefix}"
+    system "make", "clean"
```

Comment:
some temporary installation files are written under `/tmp/SourceKittent.dst`, this cleans it up.